tools/depends/configure.ac

```diff
@@ -458,10 +452,12 @@ case $use_platform in
      platform_ldflags="-lpthread"
      ;;
   tvos)
-    if test "$platform_os" != "darwin_embedded"; then
-      AC_MSG_ERROR([$use_platform is only supported on the darwin_embedded platform])
-    fi
-    ;;
+     platform_cflags+=" -fembed-bitcode"
+     platform_cxxflags+=" -fembed-bitcode"
+     if test "$platform_os" != "darwin_embedded"; then
+       AC_MSG_ERROR([$use_platform is only supported on the darwin_embedded platform])
+     fi
+     ;;
```

Comment:
Why change the indentation level?

Reply:
Indentation level was just cleaned up to match the other platforms above it.